### DISCUSSION TITLE
What 🇺🇸 state is Baton Rouge located in?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1197] [Galaxy Trivia] [answered]
**Bonus:** What country was Pope Francis born in?

**Comment by peterbaccaro** [comment, marked as answer]
Baton Rouge is located in the US state of **Louisiana**.

Pope Francis was born in Buenos Aires, **Argentina**.

**Comment by Sajansharma0017** [comment]
United States of Louisiana.
